.github/scripts/install-libraries.sh

```diff
@@ -2,38 +2,17 @@
 
 source `dirname ${BASH_SOURCE[0]}`/config.sh
 
-MPFR_VERSION=mpfr-4.1.0
-GMP_VERSION=gmp-6.2.1
-MPC_VERSION=mpc-1.2.1
-ISL_VERSION=isl-0.24
-WGET_OPTIONS="-nc -P $DOWNLOADS_PATH"
-
 echo "::group::Install libraries"
-    mkdir -p $DOWNLOADS_PATH
-
-    # Download packages
-    wget $WGET_OPTIONS https://gcc.gnu.org/pub/gcc/infrastructure/$MPFR_VERSION.tar.bz2
-    wget $WGET_OPTIONS https://gcc.gnu.org/pub/gcc/infrastructure/$GMP_VERSION.tar.bz2
-    wget $WGET_OPTIONS https://gcc.gnu.org/pub/gcc/infrastructure/$MPC_VERSION.tar.gz
-    wget $WGET_OPTIONS https://gcc.gnu.org/pub/gcc/infrastructure/$ISL_VERSION.tar.bz2
 
-    # Extract everything
-    cd $SOURCE_PATH
-    for f in $DOWNLOADS_PATH/*.tar*; do tar xf $f --skip-old-files; done
+    # Download and install packages
+    cd $SOURCE_PATH/$GCC_VERSION
+    ./contrib/download_prerequisites
 
     # Symbolic links for binutils dependencies
     cd $SOURCE_PATH/$BINUTILS_VERSION
-    ln -sf `ls -1d ../mpfr-*/` mpfr
-    ln -sf `ls -1d ../gmp-*/` gmp
-    ln -sf `ls -1d ../mpc-*/` mpc
-    ln -sf `ls -1d ../isl-*/` isl
+    ln -sf $SOURCE_PATH/$GCC_VERSION/gmp gmp
+    ln -sf $SOURCE_PATH/$GCC_VERSION/mpfr mpfr
```

Comment:
The dependencies should be linked both to binutils and GCC repo. I guess this is the reason of Cygwin fail.